gnomad_qc/v4/annotations/README.md

```diff
@@ -0,0 +1,61 @@
+# Additional annotations
+
+## CADD
+
+CADD (Rentzsch et al., Nucleic Acids Research, 2018), a score predicting deleteriousness for both SNVs and indels. Listed on the browser are the CADD Phred scores, which range from 1 to 99, based on the rank of each variant relative to all possible 8.6 billion substitutions in the human reference genome. Higher scores are predicted to be more deleterious/damaging. Variants in gnomAD v3.1.1 were annotated with CADD v1.6. Pre-computed CADD scores are available [here](https://cadd.gs.washington.edu/download).
+
+For gnomAD v4 release, we used the latest CADD v1.6 post-release 1 (released on Mar 22, 2021) to compute a score for the new indels (~32,561,253). The following Hail commands were used to generate 1000 small VCF files: 
```

Comment:
is there a more specific version number or is it called "post-release 1" ?

Reply:
It's called "post-release 1",  still in v1.6, no major changes I guess.